mkdirp/test.ts

```diff
@@ -0,0 +1,30 @@
+import { args, lstat, removeAll, FileInfo } from 'deno'
+import { test, assert } from 'https://deno.land/x/testing/testing.ts'
+import mkdirp from './mkdirp.ts'
+
+const root_dir: string = args[1] || './mkdirp_test_root_dir'
+
+test(async function nestedDirs () : Promise<void> {
+  const leaf_dir: string = `${root_dir}/levelx/levely`
+  await mkdirp(leaf_dir)
+  const info: FileInfo = await lstat(leaf_dir)
+  assert(info.isDirectory())
+  await removeAll(root_dir)
+})
```

Comment:
There should be semicolons at the end of these statements. Try running "./format.py" in the root.